bin/oref0-online.sh

```diff
@@ -13,26 +13,47 @@ main() {
     fi
     echo -n "At $(date) my local IP is: "
     print_local_ip wlan0
-    print_local_ip bnep0
-    echo
-    print_wifi_name
-    if check_ip; then
-        # if we are back on wifi (and have connectivity to checkip.amazonaws.com), shut down bluetooth
-        stop_hotspot
+	if ifconfig | egrep -q "bnep0" >/dev/null; then
+		print_local_ip bnep0
+	fi
+	echo -n "At $(date) my Public IP is: "
+    check_ip
+    #echo
+	if [[ $(ip -4 -o addr show dev wlan0 | awk '{split($4,a,"/");print a[1]}') = $(print_local_ip wlan0) ]]; then
```

Comment:
What do these checks do?  Are you just trying to check whether `print_local_ip wlan0` returns a non-empty string?  If so, `if ! [[ -z $(print_local_ip wlan0) ]]; then` would do the trick.

Reply:
this is determining if we are connected to bluetooth or Wifi. and only telling us the name of the network, or the Mac Id that it's connected too.